styles/widgets/generic/color-schemes/dark/generic.dark.less

```diff
@@ -1580,6 +1580,12 @@
 * @group treelist
 */
 @treelist-spin-icon-color: darken(@base-icon-color, 38.5%);
+/**
+* @name 152. Selection spin icon color
+* @type color
+* @group treelist
+*/
+@treelist-selection-spin-icon-color: darken(@base-icon-color, 38.5%);
```

Comment:
The duplicate code is not good. Change code to the following:

@treelist-selection-spin-icon-color: @treelist-spin-icon-color;